benchmark/kernels/fused_moe_triton/tuning_fused_moe_triton.py

```diff
@@ -413,71 +413,76 @@ def save_configs(
 def main(args: argparse.Namespace):
     print(args)
 
+    assert args.ep_size in [
+                1,
+                args.tp_size,
+            ], "The expert parallel size must be 1 or the same as the tensor parallel size"
```

Comment:
why do we need this assertion

Reply:
This is a deprecated command. `ep_size` < `tp_size` was later supported in https://github.com/sgl-project/sglang/pull/8590.